binding/rust/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "pv_rhino"
+version = "0.0.1"
```

Comment:
Rhino is 1.6.x right now

Reply:
Yup, left it as `0.0.1` for the test Publish for now